controllers/apply/awards-for-all/fields.js

```diff
@@ -1731,9 +1800,75 @@ module.exports = function fieldsFor({ locale, data = {} }) {
             })
         }),
         organisationType: organisationTypeField(),
+        organisationSubTypeStatutoryBody: {
```

Comment:
My thinking here was that we could add more fields like this if other org types needed sub-types, and just give them the same field name (eg. `organisationSubType`).